src/lib/holocene/combobox/combobox.svelte

```diff
@@ -113,12 +113,13 @@
       closeList();
     } else {
       openList();
-      inputElement.focus();
     }
   };
 
   const openList = () => {
     $open = true;
+    inputElement.focus();
```

Comment:
QQ: Would `autofocus` work here? Alternatively? I wonder if a Svelte directive that was reusable would be even better?

```ts
import type {Action} from 'svelte/actions';

export const focus: Action = (node: HTMLInputElement, shouldFocus: boolean | undefined) => {
  // Focus on mount if no value is passed or we are told to focus.
  if (shouldFocus === undefined || shouldFocus) {
    node.focus();
    node.select();
  }

  return {
    update: (shouldFocus: boolean | undefined) => {
      if (shouldFocus) {
        node.focus();
        node.select();
      }
    },
    destroy: () => {},
  }
}
```

Reply:
And then you could just do `use:focus` or `use:focus={$open}`

Reply:
I don't think autofocus would work, because I think it would focus the input on mount, whereas we only want to focus the input when the button to open the options is clicked. And In a component which such a specific interaction, I don't know that there's any real value in creating/using a more generic directive here. If we find opportunity for re-use later on we can certainly create one.